[2023-10-17T11:19:42.609+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [queued]>
[2023-10-17T11:19:42.619+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [queued]>
[2023-10-17T11:19:42.619+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:42.635+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-31 00:00:00+00:00
[2023-10-17T11:19:42.642+0000] {standard_task_runner.py:57} INFO - Started process 735 to run task
[2023-10-17T11:19:42.644+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-31T00:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpf8_2sl35']
[2023-10-17T11:19:42.645+0000] {standard_task_runner.py:85} INFO - Job 155: Subtask waiting_for_data
[2023-10-17T11:19:42.691+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:42.758+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-31T00:00:00+00:00'
[2023-10-17T11:19:42.767+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:42.767+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:42.767+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:42.768+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:42.774+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210531T000000, start_date=20231017T111942, end_date=20231017T111942
[2023-10-17T11:19:42.817+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:42.830+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:20.751+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [queued]>
[2023-10-18T10:43:20.758+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [queued]>
[2023-10-18T10:43:20.759+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:20.768+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-31 00:00:00+00:00
[2023-10-18T10:43:20.772+0000] {standard_task_runner.py:57} INFO - Started process 1704 to run task
[2023-10-18T10:43:20.774+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-31T00:00:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpk7g_ybxq']
[2023-10-18T10:43:20.775+0000] {standard_task_runner.py:85} INFO - Job 154: Subtask waiting_for_data
[2023-10-18T10:43:20.810+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:20.862+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-31T00:00:00+00:00'
[2023-10-18T10:43:20.874+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:20.877+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210531T000000, start_date=20231018T104320, end_date=20231018T104320
[2023-10-18T10:43:20.885+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 154 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1704)
[2023-10-18T10:43:20.906+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:20.927+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:43:43.355+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [queued]>
[2023-10-18T11:43:43.365+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [queued]>
[2023-10-18T11:43:43.366+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:43:43.388+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-31 00:00:00+00:00
[2023-10-18T11:43:43.403+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-31T00:00:00+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpk1qfrl01']
[2023-10-18T11:43:43.404+0000] {standard_task_runner.py:85} INFO - Job 262: Subtask waiting_for_data
[2023-10-18T11:43:43.397+0000] {standard_task_runner.py:57} INFO - Started process 1057 to run task
[2023-10-18T11:43:43.478+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-31T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:43:43.596+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-31T00:00:00+00:00'
[2023-10-18T11:43:43.612+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:43:43.612+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:43:43.612+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:43:43.612+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:43:43.623+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210531T000000, start_date=20231018T114343, end_date=20231018T114343
[2023-10-18T11:43:43.669+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:43:43.704+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
